buildconfig/Jenkins/jenkins-agent.sh

```diff
@@ -112,6 +124,12 @@ echo "starting ..."
 if [ -z "${JAVA}" ]; then
   JAVA=`which java`
 fi
-JAVA_ARGS="${PROXY_ARGS} -jar ${JAR_FILE} -jnlpUrl ${AGENT_URL} -secret ${SECRET}"
+
+if [[ "$OSTYPE" == "darwin"* ]] && [[ $(uname -m) == 'arm64' ]]; then
```

Comment:
Can we remove this if statement, assigning a value to `JAVA_ARGS` in the previously added if statement?